sdk/src/test/java/com/uber/sdk/rides/client/internal/RetrofitUberRidesClientIntegrationTest.java

```diff
@@ -321,6 +323,84 @@ private static void assertLocalizedProducts(RecordedRequest request, Response<Pr
     }
 
     @Test
+    public void onGet_whenSyncRideRequestReceipt_shouldSucceed() throws Exception {
```

Comment:
Just delete this and the associated resource.  Since we're using recorded responses all this really does is test that a GET works and that the json parsing of a fixed schema we provide, so generally we haven't been adding tests for every single GET since this test won't highlight a breaking backend change anyway.